validator/Cargo.toml

```diff
@@ -1,3 +1,18 @@
+# Copyright 2018 Intel Corporation
```

Comment:
Also includes minor changes from Cargill.

Reply:
How do you propose we document Cargill's changes. Should we include an additional Copyright line in the License header?

Reply:
Yes, an additional line is fine. Long-term we are going to replace this with a different copyright header but still waiting on that to be finalized (discussion in #sawtooth-governance).  The Cargill text is "Copyright 2018 Cargill Incorporated".